app/scripts/components/common/map/controls/aoi/index.tsx

```diff
@@ -47,33 +52,38 @@ export default function DrawControl(props: DrawControlProps) {
         },
         ...props
       });
-      return control.current;
+      return control;
     },
-    ({ map }: { map: any }) => {
-      map._drawControl = control.current;
+    ({ map }) => {
+      mapRef.current = map as ExtendedMapRef;
```

Comment:
Where is `mapRef` used? Am I missing anything?

Reply:
Removed